src/utils/options.js

```diff
@@ -52,8 +38,17 @@ const baseSchema = {
                 },
                 "type": {
                     "type": "string",
-                    "enum": ["nominal", "quantitative", "tree"],
+                    "enum": ["nominal", "quantitative", "url", "tree"],
```

Comment:
We should add this to the "type" for `sortInfo` as well, since right now when you try to sort the links it throws an error since "url" is not an allowed value.

Reply:
Thats right. I just added it.

Reply:
Great